### PR TITLE
MAUI-2935-Add UG For Until And Keyboard Navigation

### PR DESCRIPTION
**Pull request by Rishvanth-Kumaresan**
Added UG content for recurrence rule UNTIL property and keyboard navigation.

### CODE REVIEW

File: MAUI/Scheduler/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<td>
+Ctrl + Alt + 5 => TimelineDayView<br/><br/></td><td>
+Moves the view to Timeline day view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 6 => TimelineWeekView<br/><br/></td><td>
+Moves the view to Timeline week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 7 => TimelineWorkWeekView<br/><br/></td><td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:119]: Add Note , Restriction

File: MAUI/Scheduler/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the same time slot on the next day.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the same time slot on the previous day.<br/><br/></td></tr>
+<tr>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:21]: Mention next or previous date if RTL and LTR

File: MAUI/Scheduler/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+<table>
+<tr>
+<td>
+Ctrl + Alt + 1 => DayView<br/><br/></td><td>
+Moves the view to day view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 2 => WeekView<br/><br/></td><td>
+Moves the view to week view.<br/><br/></td></tr>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:100]: Add MAC platform keys for view navigation

File: MAUI/Scheduler/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+## Keyboard navigation
+
+The `SfScheduler` supports selection using keyboard interactions.
+
+#### Day, Week and WorkWeek views
+
+<table>
+<tr>
+<td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:11]: Add toc file for main title
https://github.com/syncfusion-content/maui-docs/blob/development/maui-toc.html

File: MAUI/Scheduler/accessibility.md (hunk 3 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:100]: Added MAC platform keys for view navigation.

File: MAUI/Scheduler/accessibility.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+Ctrl + Alt + 6 => TimelineWeekView<br/><br/></td><td>
+Moves the view to Timeline week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 7 => TimelineWorkWeekView<br/><br/></td><td>
+Moves the view to Timeline work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 8 => TimelineMonthView<br/><br/></td><td>
+Moves the view to Timeline month view.<br/><br/></td></tr>
```
[the remark this answers is not in the document]
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:124]: Added header for key navigation.

File: MAUI/Scheduler/accessibility.md (hunk 2 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:21]: Mentioned next or previous date for RTL and LTR as discussed.

File: MAUI/Scheduler/accessibility.md (hunk 4 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:11]: Updated toc file for accessibility.

File: MAUI/Scheduler/accessibility.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Accessibility Support in .NET MAUI Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selection using keyboard interactions.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:14]: The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interaction in WinUI, MAC Catalyst, and iOS platforms.

File: MAUI/Scheduler/accessibility.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+</table>
+
+N> There is a framework issue with unable to remove or modify  default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193).
+
+#### MAC
+
+<table>
+<tr>
+<td>
+Ctrl + 1 => DayView<br/><br/></td><td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:177]: Its Command key, not Ctrl key

File: MAUI/Scheduler/accessibility.md (hunk 1 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:156]: Added framework known issues regarding keyboard.

File: MAUI/Scheduler/accessibility.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+Moves the view to Timeline month view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 9 => AgendaView<br/><br/></td><td>
+Moves the view to Agenda view.<br/><br/></td></tr>
+</table>
+
+N> There is a framework issue with unable to remove or modify  default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193).
+
+#### MAC
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:170]: #### MAC and iOS

File: MAUI/Scheduler/accessibility.md (hunk 7 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md]: Updated as per discussion.

File: MAUI/Scheduler/accessibility.md (hunk 8 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md]: Updated as MAC and iOS.

File: MAUI/Scheduler/accessibility.md (hunk 6 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md]: Updated as mentioned.

File: MAUI/Scheduler/accessibility.md [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+### Timeline views
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the next (LTR) or previous (RTL) time slot of the currently selected time slot. <br/><br/></td></tr>
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/accessibility.md:53]: Can we mention selection change triggers for each navigation from the arrow key?

**Review by Nijamudeen-Mohamed — APPROVED** [review]
Fine

File: MAUI/Scheduler/accessibility.md [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Accessibility Support in .NET MAUI Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interaction in WinUI, MAC Catalyst, and iOS platforms.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/accessibility.md:14]: The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interaction in WinUI, MAC Catalyst, and iOS platforms. SelectionChanged event triggers for cell selection changed. 


Add API link - SelectionChanged

**Comment by Rishvanth-Kumaresan** [comment]
rebuild

File: MAUI/Scheduler/accessibility.md (hunk 9 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:56]: Added selection trigger for cell selection with API link.

File: MAUI/Scheduler/accessibility.md (hunk 10 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/Scheduler/accessibility.md:14]: Modified with API link as per discussion.

File: MAUI/Scheduler/accessibility.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: Accessibility in .NET MAUI SfScheduler control | Syncfusion
+description: Learn here about Accessibility support with the Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements and more.
```
Comment (OnyimbiNelson) [inline MAUI/Scheduler/accessibility.md:4]: description: Learn here about Accessibility support with the Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more.

File: MAUI/Scheduler/accessibility.md [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Accessibility Support in .NET MAUI Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interaction in WinUI, MAC Catalyst, and iOS platforms.
```
Comment (OnyimbiNelson) [inline MAUI/Scheduler/accessibility.md:14]: The `SfScheduler` supports selecting the scheduler date time, appointments, and view navigation using keyboard interactions in WinUI, MAC Catalyst, and iOS platforms.

File: MAUI/Scheduler/accessibility.md [hunk 13]
```diff
@@ -0,0 +1,231 @@
...
+<td>
+Ctrl + Alt + 8 => TimelineMonthView<br/><br/></td><td>
+Moves the view to Timeline month view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 9 => AgendaView<br/><br/></td><td>
+Moves the view to Agenda view.<br/><br/></td></tr>
+</table>
+
+N> There is a framework issue with unable to remove or modify  default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193).
```
Comment (OnyimbiNelson) [inline MAUI/Scheduler/accessibility.md:170]: N> There is a framework issue with unable to remove or modify the default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193):

File: MAUI/Scheduler/accessibility.md [hunk 14]
```diff
@@ -0,0 +1,231 @@
...
+<tr>
+<td>
+Tab<br/><br/></td><td>Move to next focusable element in header.<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Move to previous focusable element in header.<br/><br/></td></tr>
+</table>
+
+N> You can also explore our [MAUI Scheduler example](https://github.com/syncfusion/maui-demos) to knows how to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.
```
Comment (OnyimbiNelson) [inline MAUI/Scheduler/accessibility.md:231]: N> You can also explore our [MAUI Scheduler example](https://github.com/syncfusion/maui-demos) to know how to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.

**Pull request merged**